packages/decdk/test/fixture/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "inlineSourceMap": true,
```

Comment:
This file needs to be removed and ignored.

Reply:
done